bot_components/slack_comm.py

```diff
@@ -8,7 +8,7 @@
 """
 import json
 
-import bot_components
+from .. import bot_components
```

Comment:
@bboe I'm having an utter **NIGHTMARE** with the relative imports. Getting things working with the Duo plugin and the Travis plugin is causing a bunch of havoc.

For example, the following stacktrace results:
```
  File "/hubcommander/plugins/hubcommander.py", line 11, in <module>
    from .auth_plugins.enabled_plugins import AUTH_PLUGINS
  File "/hubcommander/plugins/auth_plugins/enabled_plugins.py", line 6, in <module>
    from .duo.plugin import DuoPlugin
  File "/hubcommander/plugins/auth_plugins/duo/plugin.py", line 14, in <module>
    from bot_components.slack_comm import send_info, send_error, send_success
  File "/venv/lib/python3.5/site-packages/bot_components/slack_comm.py", line 11, in <module>
    from .. import bot_components
ValueError: attempted relative import beyond top-level package
```